File: Python Book/4. Simple Conditions/14_time_plus_15_minutes/time_plus_15_minutes.py
hours = int(input())
minutes = int(input())
add_minutes = minutes + 15
if add_minutes <= 59:
    hours += 0
    minutes = add_minutes
elif add_minutes >= 60:
    hours += 1
    minutes = add_minutes - 60

if hours >= 24:
    hours = 0
if minutes < 10:
    print(f"{hours}:0{minutes}")
else:
    print(f"{hours}:{minutes}")

File: JavaScript/Programming Basics - JavaScript/Conditional Statements/timePlus15Minutes.js
function add15Minutes (input) {
    let hour = Number(input[0]);
    let minutes = Number(input[1]);
    let allInMinutes = hour * 60 + minutes + 15;
    hour = Math.floor(allInMinutes / 60);
    if (hour === 24) {
        hour = 0;
    } 
    minutes = allInMinutes % 60;
    if (minutes < 10) {
        console.log(`${hour}:0${minutes}`);
    } else {
        console.log(`${hour}:${minutes}`);
    }
}

add15Minutes(["1", "46"]);
add15Minutes(["0", "01"]);
add15Minutes(["23", "59"]);
add15Minutes(["11", "08"]);
add15Minutes(["12", "49"]);